README.md

```diff
@@ -1,2 +1,56 @@
-# pacifica-dispatcher-example
-Pacifica Dispatcher Example Repository
+# Pacifica Dispatcher Example
+[![Build Status](https://travis-ci.org/pacifica/pacifica-dispatcher-example.svg?branch=master)](https://travis-ci.org/pacifica/pacifica-dispatcher-example)
+[![Build status](https://ci.appveyor.com/api/projects/status/eg2r1y37yvxi0b5p?svg=true)](https://ci.appveyor.com/project/dmlb2000/pacifica-example)
+[![Maintainability](https://api.codeclimate.com/v1/badges/f2dba248b1a7966e5a49/maintainability)](https://codeclimate.com/github/pacifica/pacifica-dispatcher-example/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/f2dba248b1a7966e5a49/test_coverage)](https://codeclimate.com/github/pacifica/pacifica-dispatcher-example/test_coverage)
```

Comment:
This line can get chucked, this coverage report thing depends on out dated pipelines to report... I need to investigate getting a badge for 100% coverage that isn't code climate.

Reply:
Done!